features/invoker-commands.yml

```
@@ -0,0 +1,13 @@
+name: Invoker commands
+description: The `command` and `commandfor` attributes for the `<button>` HTML element set a name and target to dispatch to an event when the button is invoked (by click or keystroke), as a declarative alternative to `addEventListener()` calls or `onclick` attribute handlers.
```

I think there's a typo here:

```suggestion
description: The `command` and `commandfor` attributes for the `<button>` HTML element set a name and target to dispatch to an element when the button is invoked (by click or keystroke), as a declarative alternative to `addEventListener()` calls or `onclick` attribute handlers.
```

But I wonder if something like this would be simpler:

```suggestion
description: The `command` and `commandfor` attributes for the `<button>` HTML element dispatch an action to an element when the button is invoked (by click or keystroke), as a declarative alternative to `addEventListener()` calls or `onclick` attribute handlers.
```

---

I like this second version a lot!